roles/custom/matrix-bridge-mautrix-whatsapp/templates/config.yaml.j2

```diff
@@ -396,9 +396,9 @@ bridge:
     relay:
         # Whether relay mode should be allowed. If allowed, `!wa set-relay` can be used to turn any
         # authenticated user into a relaybot for that chat.
-        enabled: false
+        enabled: {{ matrix_mautrix_whatsapp_relaybot_enabled | default(false) }}
```

Comment:
Judging by the setting this influences, it should probably be named `..._bridge_relay_enabled`

Reply:
Happy to change it. This name is consistent with the signal name though.